lang/java/compiler/src/main/java/org/apache/avro/compiler/specific/SpecificCompiler.java

```diff
@@ -1045,7 +1062,7 @@ public String javaSplit(String s) throws IOException {
   /**
    * Utility for template use. Escapes quotes and backslashes.
    */
-  public static String javaEscape(String o) {
+  public static String escapeForJavaString(String o) {
```

Comment:
This is an API break.
Please keep the old method as deprecated for the lifecycle of 1.12.

Reply:
Ping!